### PR TITLE
Update devenv.rst

### PR DESCRIPTION
**Pull request by filisko**

### CODE REVIEW

File: developer_manual/general/devenv.rst [hunk 1]
```diff
@@ -11,7 +11,7 @@ Please follow the steps on this page to set up your development environment.
 Set up Web server and database
 ------------------------------
 
-First `set up your Web server and database <https://docs.nextcloud.org/server/14/admin_manual/installation/index.html>`_ (**Section**: Manual Installation - Prerequisites).
+First `set up your Web server and database <https://docs.nextcloud.org/server/16/admin_manual/installation/index.html>`_ (**Section**: Manual Installation - Prerequisites).
```
Comment (skjnldsv) [inline developer_manual/general/devenv.rst]:
```suggestion
First `set up your Web server and database <https://docs.nextcloud.org/server/stable/admin_manual/installation/index.html>`_ (**Section**: Manual Installation - Prerequisites).
```

File: developer_manual/general/devenv.rst [hunk 2]
```diff
@@ -20,7 +20,7 @@ Get the source
 
 There are two ways to obtain Nextcloud sources:
 
-* Using the `stable version <https://docs.nextcloud.org/server/14/admin_manual/#installation>`_
+* Using the `stable version <https://docs.nextcloud.org/server/16/admin_manual/#installation>`_
```
Comment (skjnldsv) [inline developer_manual/general/devenv.rst]:
```suggestion
* Using the `stable version <https://docs.nextcloud.org/server/stable/admin_manual/#installation>`_
```

**Comment by filisko** [comment]
@skjnldsv that's better!

**Comment by skjnldsv** [comment]
Thanks for the pull request! :)

**Pull request merged**